### ISSUE TITLE
Tool Menu Will not display

### ISSUE THREAD

**Issue by TheDawgMiner** [Khroki/MCEdit-Unified#496]
I can not get the tools menu to display.
I don't know if it matters or helps but my laptops i7 does not support OpenGL

![where](https://cloud.githubusercontent.com/assets/14136750/9698580/3f66b052-538b-11e5-9529-a1c383a5431e.png)

**Comment by Khroki** [comment]
What's your graphics card? I'm not aware of any card recent enough to be on an i7 laptop and not have Opengl, besides MCEdit wouldn't start if you didn't. If you don't know, hold down the windows key and press R, then in the box that pops up type dxdiag and press enter. Your graphics card is under Display 1.

Try making sure your graphics drivers are up to date. You can get drivers from the video card manufacturer's website (Or Intel's if it's integrated).

**Comment by Rubisk** [comment]
Window size is to big. You can see the outline of the selected select tool at the very bottom. 

Try resizing the window using windows key + arrow left/right

**Comment by Khroki** [comment]
That looks like the windows taskbar to me rubisk, but yeah, not sure what's wrong exactly, Usually this problem is accompanied by a much older video card. You could try the win 8 drivers.
32bit: https://downloadcenter.intel.com/download/24970/Intel-HD-Graphics-Driver-for-Windows-7-8-8-1-32bit
64bit: https://downloadcenter.intel.com/download/24971/Intel-HD-Graphics-Driver-for-Windows-7-8-64-bit

**Comment by TheDawgMiner** [comment]
Well just downloaded & installed 2.0.0alpha and got this:
No info on Intel site lists the OpenGL for this chip
http://www.cpu-world.com/CPUs/Core_i7/Intel-Core%20i7%20Mobile%20I7-2630QM.html

list it as 3.1

**Comment by TheDawgMiner** [comment]
Yes, the image on twitter had part of my taskbar showing & possibly part of a screenshot from 1.7.? when I asked about "buttons".

**Comment by Khroki** [comment]
It only needs to support Opengl 1.1, but some intel chips have poor support for older opengl versions, driver updates often help which is why I posted the links. If this isn't what you mean I am having trouble parsing your responses since they're mixed in with emails.

**Comment by TheDawgMiner** [comment]
Sorry was replying via email. The links to the drivers are the ones that have been installed for a couple months now

**Comment by Khroki** [comment]
Ah ok, hmm, Well I figure it's one of two things, either the intel stock drivers aren't fully supported on your laptop, not uncommon. If this is the case you can try downgrading to acer's latest official driver. Alternatively your card may only have partial opengl support/ quirks in it's opengl implementation, this is seen more in very old cards and a BIOS update may fix it.

**Comment by Khroki** [comment]
If neither of those fix it I'm not really sure what could be the cause outside of acer's graphics implementation. Only machines I've personally had that had strange opengl behavior were 15 year old ATI/NVidia video cards or pre-HD series Intel cards.

**Comment by TheDawgMiner** [comment]
I'll give it a go but there latest driver is from 2012/06/25

**Comment by Khroki** [comment]
Yeah :/ I try to always avoid Intel-gfx laptops whenever possible, even if the alternative is a really bad nvidia/amd video adapter because of stuff like this, the Intel cards don't age well.

**Comment by TheDawgMiner** [comment]
Well, being disabled I had to settle for the best the pawnshop had to offer lol

**Comment by Khroki** [comment]
Although if anyone has any ideas as to why the compass works and not the toolbars, we may be able to do some kind of workaround.

**Comment by TheDawgMiner** [comment]
my current "workaround" is 1.7 lol

**Comment by Khroki** [comment]
Hmm, when I started building I changed the libraries to a faster but newer opengl implementation, that's likely the problem.

Also I can appreciate the laptop thing, I use a 7 year old IBM laptop I got for free. It had an ATI card in it though so it doesn't give me much trouble (After having to tweak desktop ones).

**Comment by TheDawgMiner** [comment]
Update: The Acer drivers did not work either.

**Comment by Khroki** [comment]
Bios update help at all?

**Comment by TheDawgMiner** [comment]
I'd have to check but Acer gave up on this model in 2012 so I believe I have the lastest

**Comment by TheDawgMiner** [comment]
Grrr. Ascer is saying I have to flash the bios in DOS from a USB. I don't have a 16GB usb that FlashBoot wants.

**Comment by Khroki** [comment]
Btw, if you press the number keys does mcedit change tools?

**Comment by Khroki** [comment]
and if so do any of their interfaces work?

**Comment by TheDawgMiner** [comment]
You can't press the numbers if you can no longer get the program to load #sarcasm  but TY anyway.

**Comment by Khroki** [comment]
lol, the old drivers were worse weren't they.

**Comment by Khroki** [comment]
Just revert them and it should go back to partially working.

**Comment by TheDawgMiner** [comment]
its hit or miss, with the old drivers I get the OpenGL crash about 90% of the time.

**Comment by Khroki** [comment]
Looking at this further rubisk is right, you can see some of the toolbar at the bottom. Guessing multi-monitor support is just wonky or somesuch. Either way your MCEdit window is too large and is cutting everything off. Upgraded window handling is in the works.
